scripts/Util.py

```diff
@@ -736,12 +736,15 @@ def getProps(self, process, current):
             if isinstance(process, IceProcess):
                 props["Ice.Warn.Connections"] = True
                 if self.transport:
-                    props["Ice.Default.Transport"] = self.transport
+                    useTestTransport = isinstance(process.getMapping(current), CSharpMapping) \
+                                      and not process.isFromBinDir()
+                    transportProperty = "Test.Transport" if useTestTransport else "Ice.Default.Transport"
+                    props[transportProperty] = self.transport
                 if self.protocol:
                     useTestProtocol = isinstance(process.getMapping(current), CSharpMapping) \
                                       and not process.isFromBinDir()
-                    hostProperty = "Test.Protocol" if useTestProtocol else "Ice.Default.Protocol"
-                    props[hostProperty] = self.protocol
+                    protocolProperty = "Test.Protocol" if useTestProtocol else "Ice.Default.Protocol"
```

Comment:
This doesn't look correct.

We don't use Ice.Default.Protocol anywhere on master, except possibly for the 3.7 Ice for Python used to run the tests (and with 3.7, Protocol meant Transport).

Reply:
Right, I think it's safe to get rid of `Ice.Default.Protocol` here.